src/index.jsx

```diff
@@ -49,27 +55,62 @@ function Form() {
         setPref('app', app);
     }, [app])
 
+    const handleSelectChange = (event) => {
+        console.log(event.target)
+        const selectedItemKey = event.target.value;
+        setApp(selectedItemKey);
+    };
+
+    const options = Object.entries(AVAILABLE_APPS).map(([key, value]) => (
+        <option value={key} key={key}>{value.title}</option>
+    ));
+
     return <Box display="flex" flexDirection="column">
         <Heading sx={{ fontSize: 2, mb: 1 }}>JupyterHub URL</Heading>
 
-        <TextInput value={hubUrl} onChange={(ev) => setHubUrl(ev.target.value)} placeholder="https://myjupyterhub.org" aria-label="JupyterHub URL" />
+        <TextInput
+            value={hubUrl}
+            onChange = {
+                (ev) => setHubUrl(ev.target.value)
+            }
+            placeholder="https://myjupyterhub.org"
```

Comment:
The URL should probably be something in the example.org domain, e.g. https://jupyter.example.org. The example.org domain exists for this purpose, while someone could theoretically purchase myjupyterhub.org and abuse its inclusion in this code. (not very likely, but possible)

Reply:
I will use https://example.edu, as most professors here, no matter what domain, will be using a edu host.